README.md

```diff
@@ -74,16 +77,44 @@ print(salut({'name': 'Alice'})); //should print Hello Alice!
 ```
 
 ### Lambdas support
+
 The library passes all the optional [lambda specs][lambda_specs] based on
-which lambdas must be treatable as arity 0 or 1 functions.
-As dart provides optional named parameters, you can pass to a given lambda
-function the `nestedContext`. In that case the current nested context will be
-given as parameter to the lambda function.
+which lambdas must be treated as arity 0 or 1 functions. As dart provides
+optional named parameters, you can pass to a given lambda function the
+`nestedContext`. In that case the current nested context will be given as
+parameter to the lambda function.
+
+## Flutter / Browser support
+
+In order to achive support on targets where `dart:mirrors` is not allowed,
+[`reflectable`](https://pub.dartlang.org/packages/reflectable) is being used.
+In such a case the user should mark the objects used by mustache4dart with
+`@MustacheContext()` and make sure that the reflectable builder is being run
+accordingly.
 
+### `reflectable` setup
 
-Developing
-----------
-The project passes all the [Mustache specs][specs].  You have to make sure
+The easiest way to get reflectable to work is by making use of
+[`build_runner`](https://pub.dartlang.org/packages/build_runner) with a
+`build.yaml` that should look like:
+
+```yaml
+targets:
+  # your package name, in my case is mustache4dart
+  mustache4dart:
```

Comment:
Note that you can do `$default` here which works for any app (it fills in the package name).